browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -1738,43 +1738,43 @@ Are you sure you want to do this?
       Swap
     </message>
     <message name="IDS_DEFI_SECURE_STORAGE" desc="Brave Wallet onboarding text">
-      DeFi &amp; Secure Crypto Storage
+      Welcome to the new Brave Wallet
     </message>
     <message name="IDS_DEFI_SECURE_STORAGE_TEXT" desc="Brave Wallet onboarding text">
-      Hold crypto in your custody. Trade assets. View Prices and portfolio performance. Invest, Borrow, and lend with DeFi.
+      Hold crypto assets in your custody and track portfolio performance. All right from the Brave privacy browser.
     </message>
     <message name="IDS_SETUP_CRYPTO" desc="Brave Wallet onboarding button text">
-      Setup Crypto
+      Get started
     </message>
     <message name="IDS_BACKUP_YOUR_WALLET" desc="Brave Wallet onboarding text">
       Backup your wallet now!
     </message>
     <message name="IDS_BACKUP_YOUR_WALLET_TEXT" desc="Brave Wallet onboarding text">
-      In the next step you will see 12 words that allow you to recover your crypto wallet.
+      In the next step you’ll see a 12-word recovery phrase, which you can use to recover your primary crypto accounts. Safe it someplace safe. Your recovery phrase is the only way to regain account access in case of forgotten password, lost or stolen device, or you want to switch wallets.
```

Comment:
you should use ' char instead of `, it will fail otherwise.

Reply:
fixed